src/main/java/com/jersey/Authorization/security/UserPermissionEvaluator.java

```diff
@@ -0,0 +1,225 @@
+package com.jersey.Authorization.security;
+
+import com.jersey.persistence.*;
+import com.jersey.representations.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.PermissionEvaluator;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.stereotype.Component;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.StringTokenizer;
+
+@Component
+public class UserPermissionEvaluator implements PermissionEvaluator {
```

Comment:
Hey, do we need such a big class with so many methods. I dont see other methods other than hasPermission() being used.